src/web/routes/application/check/create-request-body.test.js

```diff
@@ -36,7 +36,8 @@ test('create claim body', (t) => {
     addressLine2: '221 Baker street',
     townOrCity: 'London',
     postcode: 'aa1 1ab',
-    phoneNumber: '07700 900000'
+    phoneNumber: '07700 900000',
+    formattedPhoneNumber: '+447700900645'
```

Comment:
It would make sense if the `phoneNumber` and `formattedPhoneNumber` we actually the same number apart from formatting.

Reply:
Done.